### DISCUSSION TITLE
How is the reporting data sent to peppol?

### DISCUSSION THREAD

**Discussion by AlexandruCiobanu** [phax/peppol-reporting#3] [Q&A] [answered]
I am investigating the peppol reporting lib and I was wondering if the actual send of the report to peppol is supported or would I have to do that explicitly.

**Comment by phax** [comment, marked as answer]
Hi @AlexandruCiobanu,

If I may quote from the main readme:

> This library does not deal with the transmission of Reports.
> That needs to be done with [phase4](https://github.com/phax/phase4) or another AS4 solution.
> See the [phase4 Wiki](https://github.com/phax/phase4/wiki/Profile-Peppol#special-handling-for-peppol-reporting) for detailed guidance on integration with this project.

:)

**Reply by alessandrolorenzetti** [reply]
Hi, I wonder if you know what the corner 4 C4 code (iso6523-actorid-upis) is to which the reports must be sent?

**Reply by phax** [reply]
This was circulated in the Service Provider Operational Guideline document via email - I don't have it at hand but I know that the production ID has not yet been setup as this is my task :-/

**Comment by alessandrolorenzetti** [comment]
I would like to take advantage of your availability and your expertise. 
I currently use a Domibus Access Point to connect to the Peppol network. 
I don't understand whether the original Sender Country (COUNTRY_C1) concept is managed by Domibus or not. 
I'm worried, because if Country_C1 isn't transmitted, the Peppol Reporting concepts won't work, I think.
I am aware that I am asking a lot, I would like a little help.
If Domibus is not the right solution I have to define a possible migration to another solution, for example phase4

**Reply by phax** [reply]
Well, I don't know Domibus well enough to give a real suggestion. It depends on, what software component creates the so called "SBDH" (Standard Business Document Header) for you.
If Domibus does this part, then I recommend that you contact the EC as the vendor and ask, if and how that can be resolved. If you create the SBDH yourself, then you simply need to add the field manually.

Proposing a different solution is difficult, because I don't know your requirements, infrastructure, usage etc. https://peppol.org/tools-support/links-to-software/ lists known AP solutions. KoSIT as the German Peppol Authority also published a document comparing the known Open Source AP solutions: https://xeinkauf.de/app/uploads/2023/09/Comparison_of_AS4_solutions_for_Peppol.pdf
